crates/ruma/src/lib.rs

```diff
@@ -87,9 +87,9 @@ pub use ruma_identifiers::{
     device_id, device_key_id, event_id, matrix_uri, mxc_uri, room_alias_id, room_id,
     room_version_id, server_name, server_signing_key_id, user_id, ClientSecret, DeviceId,
     DeviceKeyAlgorithm, DeviceKeyId, DeviceSignatures, DeviceSigningKeyId, EntitySignatures,
-    EventEncryptionAlgorithm, EventId, KeyId, KeyName, MxcUri, RoomAliasId, RoomId, RoomOrAliasId,
-    RoomVersionId, ServerName, ServerSignatures, ServerSigningKeyId, SessionId, Signatures,
-    SigningKeyAlgorithm, TransactionId, UserId,
+    EventEncryptionAlgorithm, EventId, KeyId, KeyName, MatrixToUri, MxcUri, RoomAliasId, RoomId,
```

Comment:
Can we re-export the module too, so one can refer to `ruma::matrix_uri::MatrixId` instead of `ruma::identifiers::matrix_uri::MatrixId`?

Reply:
It's already there 🙂